src/plugins/kibana_react/public/context/types.ts

```diff
@@ -22,8 +22,6 @@ import { CoreStart } from '../../../../core/public';
 import { KibanaReactOverlays } from '../overlays';
 import { KibanaReactNotifications } from '../notifications';
 
-export { CoreStart };
```

Comment:
This basically made the whole app explode due do module isolation. We really need to add an eslint rule for that kind of thing (never export types imported from other modules). I created a ticket for this.

Reply:
Why did the whole app "explode"? I think we are re-exporting types throughout Kibana, why do you want to forbid that?

Reply:
Something caused by the module isolation we are using for TS compiler. The whole thread is in the operation channel: https://elastic.slack.com/archives/C5UDAFZQU/p1571414156140200. 

But the summary is:
> We use babel to transpile the typescript which emulates the TS compiler behaviour in the isolatedModules mode. In that mode we cannot re-export types, because that would cause circular problems when generating the final code, which finally will endup in requiring things not available